### PR TITLE
update icon: gatsby (original, original-wordmark, plain-wordmark)

### PR DESCRIPTION
**Pull request by weh**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features

- Replace gatsby plain by alias
- Add newlines

**This PR closes NONE**

related to #2001

### CODE REVIEW

**Review by Snailedlt — APPROVED** [review]
Looks good to me, thank you for another great contribution! 💯 
image
image
image

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Actually, upon closer inspection it looks like there's a bit of padding on the sides (left and right) of the wordmark versions:
image
image

This padding should be removed. Other than that everything looks perfect!

File: icons/gatsby/gatsby-original-wordmark.svg [hunk 1]
```diff
@@ -1 +1 @@
...
... 0 1 5.039 2.422l2.82-2.016c-1.813-2.32-4.633-3.832-7.758-3.832h-.203a9.725 9.725 0 0 0-9.777 9.778 9.725 9.725 0 0 0 9.777 9.777h.203a9.721 9.721 0 0 0 9.774-9.777v-.403Zm19.051-.504c-1.007-1.308-2.418-2.113-4.132-2.113-3.024 0-5.442 2.922-5.442 6.45 0 3.526 2.418 6.448 5.442 6.448 1.714 0 3.226-.906 4.234-2.316v1.613h3.324V61.43h-3.426Zm-2.722 7.563a3.399 3.399 0 0 1-3.426-3.43 3.398 3.398 0 0 1 3.426-3.426 3.399 3.399 0 0 1 3.43 3.426c0 1.918-1.614 3.43-3.43 3.43Zm34.57-9.778c-1.613 0-3.125.805-4.133 2.118v-9.778h-3.324v19.856h3.324v-1.614c1.008 1.41 2.52 2.317 4.235 2.317 3.023 0 5.441-2.922 5.441-6.45 0-3.527-2.52-6.449-5.543-6.449Zm-1.309 9.778a3.399 3.399 0 0 1-3.425-3.43 3.398 3.398 0 0 1 3.425-3.426 3.398 3.398 0 0 1 3.426 3.426c-.101 1.918-1.613 3.43-3.426 3.43ZM85.47 57.398h-3.125v4.032H81.03v2.722h1.211v9.07h3.227v-9.07h2.52V61.43h-2.52Zm7.66 7.961c-.91 0-1.313-3.527 2.82-1.105l1.614-1.918s-3.93-2.82-7.258 0c-2.418 3.527 1.11 5.242 1.11 5.242l2.116.906s2.922 1.313 0 2.825c-2.418.097-3.226-1.614-3.226-1.614l-1.614 2.016s4.336 4.434 8.973 0c2.418-5.746-3.227-5.746-4.535-6.352Zm30.84-3.93-3.426 6.755-3.629-6.754h-3.832l5.543 10.28-4.434 8.063h3.93L127.7 61.43Zm0 0"/></svg>
```
Comment (Snailedlt) [inline icons/gatsby/gatsby-original-wordmark.svg:1]: optimize using SVGO
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
  <path fill="#64328b" d="M16.832 47.016C7.559 47.016 0 54.578 0 63.848 0 73.12 7.559 80.68 16.832 80.68s16.832-7.559 16.832-16.832c0-9.27-7.559-16.832-16.832-16.832ZM3.426 63.949l13.406 13.406c-7.457 0-13.406-6.046-13.406-13.406Zm16.328 13.004L3.73 60.824c1.41-5.945 6.653-10.379 13.102-10.379 4.332 0 8.266 2.117 10.684 5.34L25.5 57.7a10.742 10.742 0 0 0-8.77-4.535c-4.636 0-8.566 2.926-10.078 7.055 2.016 1.816 13.102 12.902 13.809 13.707 3.324-1.207 5.844-4.13 6.75-7.656h-5.742v-2.422l8.668.101c0 6.352-4.438 11.692-10.383 13.004Zm0 0"/>
  <path d="M56.64 63.746v2.824h4.739a6.405 6.405 0 0 1-5.945 4.032c-3.528 0-6.45-2.922-6.45-6.45 0-3.527 2.922-6.453 6.45-6.453a6.41 6.41 0 0 1 5.039 2.422l2.82-2.016c-1.813-2.32-4.633-3.832-7.758-3.832h-.203a9.725 9.725 0 0 0-9.777 9.778 9.725 9.725 0 0 0 9.777 9.777h.203a9.721 9.721 0 0 0 9.774-9.777v-.403Zm19.051-.504c-1.007-1.308-2.418-2.113-4.132-2.113-3.024 0-5.442 2.922-5.442 6.45 0 3.526 2.418 6.448 5.442 6.448 1.714 0 3.226-.906 4.234-2.316v1.613h3.324V61.43h-3.426Zm-2.722 7.563a3.399 3.399 0 0 1-3.426-3.43 3.398 3.398 0 0 1 3.426-3.426 3.399 3.399 0 0 1 3.43 3.426c0 1.918-1.614 3.43-3.43 3.43Zm34.57-9.778c-1.613 0-3.125.805-4.133 2.118v-9.778h-3.324v19.856h3.324v-1.614c1.008 1.41 2.52 2.317 4.235 2.317 3.023 0 5.441-2.922 5.441-6.45 0-3.527-2.52-6.449-5.543-6.449Zm-1.309 9.778a3.399 3.399 0 0 1-3.425-3.43 3.398 3.398 0 0 1 3.425-3.426 3.398 3.398 0 0 1 3.426 3.426c-.101 1.918-1.613 3.43-3.426 3.43ZM85.47 57.398h-3.125v4.032H81.03v2.722h1.211v9.07h3.227v-9.07h2.52V61.43h-2.52Zm7.66 7.961c-.91 0-1.313-3.527 2.82-1.105l1.614-1.918s-3.93-2.82-7.258 0c-2.418 3.527 1.11 5.242 1.11 5.242l2.116.906s2.922 1.313 0 2.825c-2.418.097-3.226-1.614-3.226-1.614l-1.614 2.016s4.336 4.434 8.973 0c2.418-5.746-3.227-5.746-4.535-6.352Zm30.84-3.93-3.426 6.755-3.629-6.754h-3.832l5.543 10.28-4.434 8.063h3.93L127.7 61.43Zm0 0"/>
</svg>
```

File: icons/gatsby/gatsby-plain-wordmark.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...3.832.91 5.043 2.422l2.82-2.016c-1.812-2.32-4.637-3.832-7.762-3.832h-.199a9.722 9.722 0 0 0-9.777 9.778 9.722 9.722 0 0 0 9.777 9.777h.2a9.722 9.722 0 0 0 9.777-9.777v-.403Zm18.949-.504c-1.008-1.309-2.418-2.113-4.133-2.113-3.023 0-5.441 2.922-5.441 6.449 0 3.527 2.418 6.45 5.441 6.45 1.715 0 3.227-.907 4.235-2.317v1.613h3.324V61.48h-3.426Zm-2.723 7.562a3.401 3.401 0 0 1-3.425-3.43A3.398 3.398 0 0 1 73.07 64a3.398 3.398 0 0 1 3.426 3.426 3.401 3.401 0 0 1-3.426 3.43Zm34.672-9.777c-1.613 0-3.125.805-4.133 2.117v-9.777h-3.425v19.855h3.324V71.66c1.008 1.41 2.52 2.317 4.234 2.317 3.024 0 5.442-2.922 5.442-6.45 0-3.527-2.418-6.449-5.442-6.449Zm-1.41 9.777a3.402 3.402 0 0 1-3.43-3.43 3.399 3.399 0 0 1 3.43-3.425 3.398 3.398 0 0 1 3.426 3.426 3.401 3.401 0 0 1-3.426 3.43ZM85.668 57.45h-3.223v4.031h-1.312v2.723h1.312v9.07h3.223v-9.07h2.422V61.48h-2.422Zm7.559 7.961c-.907 0-1.309-3.527 2.824-1.11l1.613-1.913s-3.934-2.82-7.258 0c-2.418 3.527 1.11 5.242 1.11 5.242l2.117.906s2.922 1.309 0 2.824c-2.422.098-3.227-1.613-3.227-1.613l-1.613 2.016s4.336 4.433 8.973 0c2.418-5.746-3.125-5.746-4.54-6.352Zm30.843-3.93-3.43 6.754-3.624-6.754h-3.832l5.543 10.282-4.332 8.062h3.828l9.574-18.344Zm0 0"/></svg>
```
Comment (Snailedlt) [inline icons/gatsby/gatsby-plain-wordmark.svg:1]: Optimize using SVGO
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
  <path fill="#64328b" d="M16.832 47.066C7.559 47.066 0 54.625 0 63.898 0 73.172 7.559 80.73 16.832 80.73s16.832-7.558 16.832-16.832c0-9.273-7.559-16.832-16.832-16.832ZM3.426 64l13.406 13.406C9.375 77.406 3.426 71.36 3.426 64Zm16.328 13L3.73 60.875c1.41-5.945 6.653-10.379 13.102-10.379 4.332 0 8.266 2.113 10.684 5.34L25.5 57.75a10.742 10.742 0 0 0-8.77-4.535c-4.636 0-8.566 2.922-10.078 7.055 2.016 1.816 13.102 12.902 13.809 13.707 3.324-1.207 5.844-4.133 6.75-7.66h-5.742v-2.419l8.668.102c0 6.348-4.438 11.691-10.383 13Zm37.09-13.203v2.824h4.738c-.906 2.317-3.227 4.031-5.95 4.031-3.527 0-6.448-2.922-6.448-6.449 0-3.527 2.921-6.453 6.449-6.453 2.02 0 3.832.91 5.043 2.422l2.82-2.016c-1.812-2.32-4.637-3.832-7.762-3.832h-.199a9.722 9.722 0 0 0-9.777 9.778 9.722 9.722 0 0 0 9.777 9.777h.2a9.722 9.722 0 0 0 9.777-9.777v-.403Zm18.949-.504c-1.008-1.309-2.418-2.113-4.133-2.113-3.023 0-5.441 2.922-5.441 6.449 0 3.527 2.418 6.45 5.441 6.45 1.715 0 3.227-.907 4.235-2.317v1.613h3.324V61.48h-3.426Zm-2.723 7.562a3.401 3.401 0 0 1-3.425-3.43A3.398 3.398 0 0 1 73.07 64a3.398 3.398 0 0 1 3.426 3.426 3.401 3.401 0 0 1-3.426 3.43Zm34.672-9.777c-1.613 0-3.125.805-4.133 2.117v-9.777h-3.425v19.855h3.324V71.66c1.008 1.41 2.52 2.317 4.234 2.317 3.024 0 5.442-2.922 5.442-6.45 0-3.527-2.418-6.449-5.442-6.449Zm-1.41 9.777a3.402 3.402 0 0 1-3.43-3.43 3.399 3.399 0 0 1 3.43-3.425 3.398 3.398 0 0 1 3.426 3.426 3.401 3.401 0 0 1-3.426 3.43ZM85.668 57.45h-3.223v4.031h-1.312v2.723h1.312v9.07h3.223v-9.07h2.422V61.48h-2.422Zm7.559 7.961c-.907 0-1.309-3.527 2.824-1.11l1.613-1.913s-3.934-2.82-7.258 0c-2.418 3.527 1.11 5.242 1.11 5.242l2.117.906s2.922 1.309 0 2.824c-2.422.098-3.227-1.613-3.227-1.613l-1.613 2.016s4.336 4.433 8.973 0c2.418-5.746-3.125-5.746-4.54-6.352Zm30.843-3.93-3.43 6.754-3.624-6.754h-3.832l5.543 10.282-4.332 8.062h3.828l9.574-18.344Zm0 0"/>
</svg>
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Great! Now we just need to re-optimize the icons (see my suggestions), and we're good to go! 🚀

**Review by lunatic-fox — APPROVED** [review]
Looks great! 👍🏼
Thank you for this contribution! 🚀

**Pull request merged**